google-cloud-pubsub/test/helper.rb

```diff
@@ -90,9 +90,9 @@ def initialize
     @messages = []
   end
 
-  def publish topic:, messages:
```

Comment:
We're changing signature of an existing method ?

Reply:
This method is stubbed for testing which was using a short hand to unstructure the hash but as we have options used now updated it to represent the actual service method signature https://github.com/googleapis/google-cloud-ruby/blob/9c727bfc54a75b399a83a83a1d843ce34d4d4748/google-cloud-pubsub-v1/lib/google/cloud/pubsub/v1/publisher/client.rb#L464

Reply:
I didn't notice it was for tests, my bad.